content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -164,6 +164,17 @@ Docker's default is the json-file logging driver so the Container Agent tries to
 
 4. Mount the YAML file into your container following the instructions in the [Docker Agent documentation][3]. For more information on setting log drivers for Docker containers, [see this documentation][4].
 
+## Logs are missing but only for some containers
```

Comment:
I'm not sure this solution is actually the most common one for the issue of `Logs are missing but only for some containers` (in most cases, I'd expect this issue to be related to misconfiguration).

Let's rename this to:
```suggestion
## Agent doesn't send logs from containers that have persisted a large volume of logs (> 1GB)
```

What do you think?